include/regs.h

```diff
@@ -58,15 +58,26 @@
 #define R_FB_FILTER_ENV_COLOR(c)                 SREG(85 + (c))
 #define R_ENABLE_FB_FILTER                       SREG(88)
 #define R_PAUSE_MENU_MODE                        SREG(94)
+#define R_CAM_XZ_OFFSET_UPDATE_RATE              OREG(2)
+#define R_CAM_Y_OFFSET_UPDATE_RATE               OREG(3)
+#define R_CAM_FOV_UPDATE_RATE                    OREG(4)
 #define R_CAM_MAX_PITCH                          OREG(5)
-#define R_CAM_DEFAULT_PITCH_UPDATE_RATE_INV      OREG(7)
+#define R_CAM_R_UPDATE_RATE_INV                  OREG(6)
+#define R_CAM_PITCH_UPDATE_RATE_INV              OREG(7)
+#define R_CAM_SLOPE_Y_ADJ_AMOUNT                 OREG(9)
```

Comment:
Kind of minor, but not sure `amount` really adds anything to the name so could probably just be omitted.

Reply:
it's not the `yAdj`, it's the `adjAmt` that goes into computing the `yAdj = Camera_CalcSlopeYAdj(...)` 8)

it was even `R_CAM_SLOPE_Y_ADJ_ADJ_AMOUNT` before

do you still think `_Y_ADJ` would be enough?

Reply:
Ok yeah, it is fine as is for now. I do kind of agree with engineer that perhaps `adjAmt` could be looked at again, though does not have to be this PR.